plugins/lookup/aws_secret.py

```diff
@@ -96,6 +104,8 @@
 from ansible.module_utils._text import to_native
 from ansible_collections.amazon.aws.plugins.module_utils.core import is_boto3_error_code
 
+from ..module_utils.ec2 import HAS_BOTO3
```

Comment:
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.ec2 import HAS_BOTO3
```

Reply:
To add a little context here: while both are technically correct we don't have an explicit style across the collection as a whole instead we tend to try and be consistent within a module